tests/components/august/mocks.py

```diff
@@ -51,6 +105,9 @@ def __init__(
         self._last_door_state_update_time_utc = dt.as_utc(
             datetime.datetime.fromtimestamp(last_lock_status_update_timestamp)
         )
+        self._api = api
+        self._access_token = access_token
+        self._locks = [MockAugustLock()]
```

Comment:
The AugustData should not know about the august lock entities.

Reply:
I'm definitely mocking the wrong thing here.  I'll do a light cleanup for now as I'm going to have to do major refactor with the switch over to DataUpdateCoordinator